### ISSUE TITLE
GENOME_ASSEMBLY waits on ASSIGN_REFERENCES because of QUAST

### ISSUE THREAD

**Issue by zachary-foster** [nf-core/pathogensurveillance#7]
Right now genome assembly does not start until references are chosen even though references are not needed for spades. We could move QUAST out of GENOME_ASSEMBLY or redo how the channels are connected inside GENOME_ASSEMBLY

**Comment by zachary-foster** [comment]
fixed by removing reference requirement of quast in the `dev` branch.
